lib/timers.js

```diff
@@ -164,56 +196,82 @@ exports._unrefActive = function(item) {
 // Appends a timer onto the end of an existing timers list, or creates a new
 // TimerWrap backed list if one does not already exist for the specified timeout
 // duration.
-function insert(item, unrefed, start) {
+function insert(item, refed, start) {
```

Comment:
Why was this all changed from setting unrefed properties to setting refed properties?

Self-edit from later: Ok I guess this can make sense with the concept of a "refCount"... I still wish the initial setting stayed the same, personally.